packages/flame_3d/lib/src/graphics/graphics_device.dart

```diff
@@ -93,6 +95,17 @@ class GraphicsDevice {
 
   void clearBindings() {
     _renderPass.clearBindings();
+    _lights.clear();
+  }
```

Comment:
I don't think `flame_3d` is currently using the `clearBindings` in the render loop is it? We should probably call it somewhere 😅  maybe by default in the `end()`?